.circleci/config.yml

```diff
@@ -61,5 +61,4 @@ jobs:
             - run:
                 name: Release
                 command: |
-                  cd packages/integration-tests
```

Comment:
Whoops, this was added accidentally - the release script is at the root of the repo.